src/repr/src/adt/interval.rs

```diff
@@ -295,29 +308,23 @@ impl Interval {
     /// Converts this `Interval`'s duration into `chrono::Duration`.
     pub fn duration_as_chrono(&self) -> chrono::Duration {
         use chrono::Duration;
-        // This can be converted into a single call with
-        // https://github.com/chronotope/chrono/pull/426
-        Duration::seconds(self.dur_as_secs() as i64)
-            + Duration::nanoseconds(self.nanoseconds() as i64)
+        Duration::days(self.days.into()) + Duration::microseconds(self.micros)
```

Comment:
Months?

Reply:
Ah, yeah--this is a little goofy. Not sure what the expectation of this function is but it might also want to instead _not_ account for days.